.azure-pipelines/azure-pipelines-osx.yml

```diff
@@ -48,6 +48,8 @@ jobs:
       export PATH=$(Build.StagingDirectory)/miniconda/bin:$PATH
       set -x -e
       conda install -n base -c conda-forge --quiet --yes conda-forge-ci-setup=2 conda-build
+      # Patch conda use a shorter testing prefix
+      pip install --upgrade git+https://github.com/chrisburr/conda-build.git@patch-1
```

Comment:
Why this? Just curious.

Reply:
This picks up my PR to `conda-build`: https://github.com/conda/conda-build/pull/3386

It uses a 78 character prefix when testing instead of 80 characters (which is used for building). This exposes bugs in my (failed) attempts to edit the paths in the PCH.

Reply:
Will you include this in the merged version?

Reply:
I see little reason not to, it will be removed next time the feedstock is rendered.

Reply:
True, okay then!